examples/datasource-streaming-backend-websocket/README.md

```diff
@@ -1,12 +1,17 @@
-# Grafana Data Source Streaming Backend Plugin example
+# Streaming WebSocket Backend Data Source Plugin for Grafana
 
-This is an example of how to implement a Grafana data source plugin with streaming backend support.
+This backend data source plugin shows how to stream data from a WebSocket.
+## Overview
 
-The plugin connects to the backend through a streaming connection and the backend establishes a connection to an external websocket server.
+The Streaming WebSocket Backend Data Source Plugin serves as a reference implementation for developers seeking to incorporate WebSocket-based services into their Grafana dashboards for streaming data scenarios.
 
-## Build
+The plugin connects to the backend through a streaming connection and the backend establishes a connection to an external WebSocket server.
 
-Build the data source plugin
+The example server sends random numbers controlled by a query parameter.
+
+## Get started
+
+1. Build the data source plugin:
 
   ```sh
 cd streaming-backend-websocket-plugin
```

Comment:
no need to add this `cd streaming-backend-websocket-plugin` we don't do it in other readmes
```suggestion

```